### PR TITLE
Adding litmuschaos project issues to LFX mentorship 2024

### PR DESCRIPTION
**Pull request by imrajdas**
#### Enhancement of litmusctl: Adding E2E Tests, CRUD Probes Commands, and Package Manager Availability

- Description: The [project](https://github.com/litmuschaos/litmusctl) aims to improve litmusctl by introducing end-to-end (E2E) tests for better release testing and adding CRUD commands for probes, addressing user needs. Additionally, it seeks to enhance user accessibility by making litmusctl available on Brew and Chocolatey package managers.
- Expected Outcome: The enhancement of litmusctl will include comprehensive E2E testing for improved reliability, the addition of CRUD commands for probes to expand functionality, and availability on Brew and Chocolatey for greater accessibility and user convenience.
- Recommended Skills: Golang, Kubernetes
- Mentor(s):
  - Vedant Shrotria (@Jonsy13, vedant.shrotria@harness.io)
  - Sarthak Jain (@SarthakJain26, sarthak.jain@harness.io)
  - Nagesh Bansal (@Nageshbansal, nageshbansal59@gmail.com)
- Upstream Issue: https://github.com/litmuschaos/litmus/issues/4405

#### Enhancing Chaos Center: Implementing E2E Test Cases and Addressing CVE Issues

- Description: This initiative focuses on augmenting the [Chaos Center](https://github.com/litmuschaos/litmus/tree/master/chaoscenter) with comprehensive end-to-end (E2E) test cases, addressing the current gap in testing capabilities. The lack of extensive E2E tests has been a challenge, especially during release cycles. The project also targets fixing identified Common Vulnerabilities and Exposures (CVEs) in the Chaos Center, enhancing the overall security and reliability of the system.
- Expected Outcome: The project aims to establish a robust E2E testing framework for the Chaos Center, significantly improving test coverage and reliability during releases. Additionally, it focuses on resolving all identified CVEs, thereby enhancing the system's security. These improvements are expected to result in more stable and secure releases, increasing user trust in the Chaos Center.
- Recommended Skills: Golang, Kubernetes
- Mentor(s):
  - Namkyu Park (@namkyu1999, lak9348@konkuk.ac.kr)
  - Shubham Chaudhary (@ispeakc0de, shubham.chaudhary@harness.io)
  - Raj Babu Das (@imrajdas, rajbabu.das@harness.io)
- Upstream Issue: https://github.com/litmuschaos/litmus/issues/4406

#### Enhancements in Chaos Center: Multiple Project Owners and Log Download API
- Description: This project focuses on two major enhancements for the [Chaos Center](https://github.com/litmuschaos/litmus/tree/master/chaoscenter). First, it aims to enable the support for multiple project owners, a feature highly requested by users. This addition will allow for more collaborative and flexible project management within the Chaos Center. Second, the project will develop an API for downloading logs, providing users with easier access to log data. Furthermore, there's a need to update the API documentation to reflect these new changes and ensure that users have the latest information for seamless integration and usage.
- Expected Outcome: The successful completion of this project will result in the Chaos Center supporting multiple project owners, fostering collaborative and efficient project management. The new log download API will enhance user experience by simplifying access to log data. Additionally, the updated API documentation will ensure that users have clear and current guidelines, supporting better utilization of the new features.
- Recommended Skills: Golang, ReactJs
- Mentor(s):
  - Saranya Jena (@Saranya-jena, saranya.jena@harness.io)
  - Sahil Kumar (@SahilKr24, sahil.kumar@harness.io)
  - Hrishav Kumar (@hrishavjha, hrishav.kumar@harness.io)
- Upstream Issue: https://github.com/litmuschaos/litmus/issues/4407

### CODE REVIEW

**Comment by aliok** [comment]
Thanks @imrajdas  

Can the other mentors approve this PR please?

cc @Jonsy13 @SarthakJain26 @Nageshbansal @namkyu1999 @ispeakc0de @Saranya @SahilKr24 @hrishavjha

I also wanted to let you know that this term we’re doing an information session for mentors. We’d like to invite all the mentors participating to attend one of these if you're able:
* North America: [Jan 29, 2024 10:30 AM (Pacific Time)](https://zoom-lfx.platform.linuxfoundation.org/meeting/99504894798?password=54a6ec2f-d649-432b-a918-acd17720f163)
* EMEA + APAC: [Jan 30, 2024 8:00 AM (UTC)](https://zoom-lfx.platform.linuxfoundation.org/meeting/93514971498?password=907d67b6-3e04-42d3-afdc-52d288122c6f)

**Review by aliok — APPROVED** [review]

**Review by SahilKr24 — APPROVED** [review]

**Review by Nageshbansal — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -345,6 +345,40 @@ We want to leverage the above for creating a plugin which will allow users to se
...
+- Upstream Issue: https://github.com/litmuschaos/litmus/issues/4405
+
+#### Enhancing Chaos Center: Implementing E2E Test Cases and Addressing CVE Issues
+
+- Description: This initiative focuses on augmenting the [Chaos Center](https://github.com/litmuschaos/litmus/tree/master/chaoscenter) with comprehensive end-to-end (E2E) test cases, addressing the current gap in testing capabilities. The lack of extensive E2E tests has been a challenge, especially during release cycles. The project also targets fixing identified Common Vulnerabilities and Exposures (CVEs) in the Chaos Center, enhancing the overall security and reliability of the system.
+- Expected Outcome: The project aims to establish a robust E2E testing framework for the Chaos Center, significantly improving test coverage and reliability during releases. Additionally, it focuses on resolving all identified CVEs, thereby enhancing the system's security. These improvements are expected to result in more stable and secure releases, increasing user trust in the Chaos Center.
+- Recommended Skills: Golang, Kubernetes
+- Mentor(s):
+  - Namkyu Park (@namkyu1999, lak9348@konkuk.ac.kr)
```
Comment (namkyu1999) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:367]:
```suggestion
  - Namkyu Park (@namkyu1999, lak9348@gmail.com)
```

**Review by namkyu1999 — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by hrishavjha — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md (hunk 1 above)
Comment (imrajdas) [reply programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md]: done

**Review by Saranya-jena — APPROVED** [review]

**Comment by imrajdas** [comment]
Hi @aliok, We got all the approval

**Pull request merged**